azure-kusto-data/azure/kusto/data/client.py

```diff
@@ -568,6 +569,7 @@ class KustoClient:
     _mgmt_default_timeout = timedelta(hours=1, seconds=30)
     _query_default_timeout = timedelta(minutes=4, seconds=30)
     _streaming_ingest_default_timeout = timedelta(minutes=10)
+    _client_server_delta = timedelta(seconds=30)

```

Comment:
I guess you can remove the 30 seconds delta from the other defaults.

Reply:
I used the Java client (src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java) as a reference. There, :30 is in addition to (1:30|4:30|10:00).

What's the desired behavior?

Reply:
I looked up how it works in the .NET SDK (Src/Common/Kusto.Cloud.Platform/Utils/ServiceModelRequestTimeout.cs), and I see that the intent is as you indicated - 1 hour for ingest, 4 minutes for queries, 10 minutes for streaming, and everything + 30 seconds client-server delta. I'll fix this both here and in the Java SDK.